docs/source/start.ipynb

```diff
@@ -46,6 +46,8 @@
    "source": [
     "EvalML has many options to configure the pipeline search. At the minimum, we need to define an objective function. For simplicity, we will use the F1 score in this example. However, the real power of EvalML is in using domain-specific [objective functions](user_guide/objectives.ipynb) or [building your own](user_guide/objectives.ipynb).\n",
     "\n",
+    "In addition, EvalML provides a number of parameters to control the search process. `max_iterations` is one of the parameters which controls the stopping criterion for the AutoML search. It indicates the maximum number of pipelines to train and evaluate. In this example, `max_iterations` is set to 5.  \n",
```

Comment:
Thanks!

Can you move this below the next line "Below EvalML utilizes "? I think it reads easier. Also I suggest you delete "In addition, "